backend/open_webui/utils/middleware.py

```
@@ -44,9 +44,9 @@
 from open_webui.utils.webhook import post_webhook
 
 
-from open_webui.models.users import UserModel
+from beyond_the_loop.models.users import UserModel
 from open_webui.models.functions import Functions
-from open_webui.models.models import Models
+from beyond_the_loop.models.models import Models
```

Ruff (0.8.2)

`beyond_the_loop.models.models.Models` imported but unused

Remove unused import: `beyond_the_loop.models.models.Models`

(F401)